velox/common/caching/SsdFile.cpp

```diff
@@ -260,6 +263,16 @@ CoalesceIoStats SsdFile::load(
 
   for (auto i = 0; i < ssdPins.size(); ++i) {
     pins[i].checkedEntry()->setSsdFile(this, ssdPins[i].run().offset());
+    // Verify that the checksum matches after we read from SSD.
```

Comment:
I think we need compatibility handling here for checkpoint file with and without checksum.